### PR TITLE
Upgrade to v0.10.0

### PR DESCRIPTION
**Pull request by thecesrom**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = shellcheck_py
-version = 0.9.0.6
+version = 0.10.0
```
Comment (asottile) [inline setup.cfg:3]: the last element is our version and should remain

**Review by asottile — COMMENTED** [review]
we don't do conventional commit garbage here.  please rebase that out and general advise follow the convention of the existing commits

File: setup.cfg (hunk 1 above)
Comment (thecesrom) [reply setup.cfg:3]: Should it be 0.10.0.0 or 0.10.0.1?

Comment (thecesrom) [reply setup.cfg]: Changed it to 0.10.0.1.

**Comment by arkinmodi** [comment]
Shellcheck v10 adds precompiled binaries for macos arm64. Adding support would fix #85.

**Comment by thecesrom** [comment]
Would it be `aarch64` or `arm64`?

```ini
    [shellcheck]
    group = shellcheck-binary
    marker = sys_platform == "darwin" and platform_machine == "aarch64"
    url = https://github.com/koalaman/shellcheck/releases/download/v0.10.0/shellcheck-v0.10.0.darwin.aarch64.tar.xz
    sha256 = bbd2f14826328eee7679da7221f2bc3afb011f6a928b848c80c321f6046ddf81
    extract = tar
    extract_path = shellcheck-v0.10.0/shellcheck
    [shellcheck]
    group = shellcheck-binary
    marker = sys_platform == "darwin" and platform_machine == "x86_64"
    url = https://github.com/koalaman/shellcheck/releases/download/v0.10.0/shellcheck-v0.10.0.darwin.x86_64.tar.xz
    sha256 = ef27684f23279d112d8ad84e0823642e43f838993bbb8c0963db9b58a90464c2
    extract = tar
    extract_path = shellcheck-v0.10.0/shellcheck
```

**Review by asottile — APPROVED** [review]

**Pull request merged**